proposals/networkservicemesh.adoc

```diff
@@ -0,0 +1,85 @@
+
+Network Service Mesh requires no changes to the CNI plugin being used, or to Kubernetes itself to function.  It runs as an additional layer of infrastructure on top of stock out of the box Kubernetes.
+
+When Network Service Mesh is installed on a K8s cluster, a developer can simply specify in an annotation on their Pod the name of the Network Service they want, such as "secure-intranet-connectivity".   Network Service Mesh will ensure that Network Service is available to their Pods (not to the cluster as a whole) without using CNI.  Standard Kubernetes Networking is still provided by CNI.  Network Service Mesh takes care to insure that it is completely non-conflicting with Kubernetes Networking.
+
+Network Service Mesh takes a true Cloud native approach to the problem space, maintaining loose coupling to the underlying Infrastructure it utilizes.  This allows it to fit naturally into Kubernetes and also be used in other environments like VIMs, Physical Netorks, or Legacy Servers.
+
+*Statement on alignment with CNCF mission*:
+
+Network Service Mesh is designed to allow a workload to dynamically request at any point in its lifecycle the L2/L3 Network Service it needs.   It makes networking itself Cloud native by loosely coupling with the existing immutable infrastrcture in K8s to deliver whatever non-standard Networking needs a workload has with minimal toil.
+
+The Network Service Mesh model is:
+* Developers ask for what Network Service they want by name
+* Developers can add metadata to their request for a Network Service with labels.
+* Developers simply get the Network Service they asked for
+
+Developers never have to know or think about 35 year old legacy Networking concepts like:
+* Subnets
+* Interfaces
+* Routes
+* IPAM
+
+*Sponsor / Advisor from TOC*: Matt Klein, Joe Beda
+
+*Unique identifier:* networkservicemesh
+
+*Prefered Maturity Level*: Sandbox
+
+*License*: Apache License v2.0
+
+*Source control repositories*: link:https://github.com/networkservicemesh[GitHub]
+
+*Initial Committers*:
+* Frederick Kautz (doc.ai)
+* Ed Warnicke (Cisco)
+* Nikolay Nikolaev (VMWare)
+
+*Infrastructure requirements (CI / CNCF Cluster)*: CI (currently use CircleCI), Packet/GKE/AKS/EKS/etc time for system tests
+
+*Communication Channels*:
+
+* Mailing List: https://groups.google.com/forum/#!forum/networkservicemesh
+* IRC: #networkservicemesh on Freenode
+* Weekly Meetings: 
+  * link:https://docs.google.com/document/d/1C9NKjo0PWNWypROEO9-Y6haw5h9Xmurvl14SXpciz2Y/edit#heading=h.rc9df0a6n3ng[Main Meeting]
+  * link:https://docs.google.com/document/d/1113nzdL-DcDAWT3963IsS9LeekgXLTgGebxPO7ZnJaA/edit#heading=h.8t1wzcxy1me6[Documentation subgroup meeting]
+  * Use case subgroup meeting: 
+
+*Issue Tracker*: link:https://github.com/networkservicemesh/networkservicemesh/issues[Github Issues]
+
+*Website*: https://networkservicemesh.io/
+
+*Release Methodology and Mechanics*: 
```

Comment:
No official release yet. The plan is to have an initial 0.1.0 ready for KubeCon EU 2019. The current process is to decide on a particular release during the WG calls and managed through a project in the Guthub Issue tracker.
The agreed formal release process is described here: https://github.com/networkservicemesh/networkservicemesh/blob/master/docs/spec/release.md.

Reply:
Will fill in :)

Reply:
Fixed in update.